### DISCUSSION TITLE
HexRD Gui Missing Plugin

### DISCUSSION THREAD

**Discussion by JoshuaWard201** [HEXRD/hexrdgui#1707] [Q&A]
Hi @psavery, I am attempting to run the HexRD gui on my computer. I followed the installation instructions outlined in the installation documentation, and am attempting to run the gui using the "hexrdgui" command. I am in the correct environment where I installed hexrd gui and when attempting to run I get the following error, "qt.qpa.plugin: Could not find the Qt platform plugin "windows" in""  This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem". I uninstalled and reinstalled hexrd gui. Deleted the environment and recreated following the same steps for installation both to no avail. Any thoughts or ideas on how to fix this? Thanks, Joshua Ward

**Comment by psavery** [comment]
Can you try re-installing Qt? Unfortunately, I think Qt moved their plugins directory at some point when they changed versions.

In your HEXRD conda environment, try running this:

```bash
conda remove pyside6
conda remove qt6-main
conda install -c conda-forge pyside6 qt6-main
```

Anaconda might also be trying to keep older versions of Qt around. The above should hopefully fix that. But completely reinstalling anaconda is likely to also fix the issue...

**Comment by psavery** [comment]
If that still doesn't work, another alternative is to download the MSI/ZIP packages and install the program from them. Here is the latest release: https://github.com/HEXRD/hexrdgui/releases/tag/1.1.0

We will have another release soon as well.
